src/Components/v2/CarouselV2.tsx

```diff
@@ -0,0 +1,265 @@
+      <>
+        <Media at="xs">
+          <SmallCarousel {...this.props} />
+        </Media>
+        <Media greaterThan="xs">
+          <LargeCarousel {...this.props} />
+        </Media>
+      </>
+    )
+  }
+}
+
+/*
+ * Returns null if no window for SSR
+ */
+const Flickity =
+  typeof window !== "undefined"
+    ? require("react-flickity-component")
+    : () => null
+
+export const LargeCarousel = (props: Props) => {
+  const [isMounted, toggleMounted] = useState(false)
+  const [currentSlideIndex, setSlideIndex] = useState(0)
+  const [lastItemVisible, setLastItemVisible] = useState(false)
+
+  useEffect(() => {
+    toggleMounted(true)
+  }, [])
+  const flicktyRef = useRef(null)
+
+  let flickity = null
+
+  const checkLastItemVisible = () => {
+    if (!flickity.slides.length) {
+      return
+    }
+    const isLastItemVisible = flickity.selectedElements.includes(
+      flickity.getLastCell().element
+    )
+    setLastItemVisible(isLastItemVisible)
+  }
+
+  if (isMounted) {
+    if (flicktyRef.current !== null) {
+      flickity = flicktyRef.current.flkty
+    }
+
+    flickity.on("select", index => {
+      setSlideIndex(index)
+      checkLastItemVisible()
+    })
+  }
+
+  const settings = {
+    draggable: false,
+    freeScroll: false,
+    wrapAround: false,
+    cellAlign: "left",
+    pageDots: false,
+    prevNextButtons: false,
+    groupCells: true,
+    contain: true,
+    ...props.settings,
+  }
+
+  const LeftArrow = () => {
+    const getFlickity = flickity
+    return (
+      <ArrowButton
+        onClick={() => {
+          getFlickity && getFlickity.previous ? getFlickity.previous() : null // check existence for tests
+        }}
+      >
+        <ChevronIcon direction="left" fill="black100" width={30} height={30} />
+      </ArrowButton>
+    )
+  }
+
+  const RightArrow = () => {
+    const getFlickity = flickity
+    return (
+      <ArrowButton
+        onClick={() => {
+          getFlickity && getFlickity.next ? getFlickity.next() : null // check existence for tests
+        }}
+      >
+        <ChevronIcon direction="right" fill="black100" width={30} height={30} />
+      </ArrowButton>
+    )
+  }
+  return (
+    <Flex
+      flexDirection="row"
+      position="relative"
+      justifyContent="space-around"
+      alignItems="center"
+      height={props.height}
+    >
+      {(currentSlideIndex !== 0 || settings.wrapAround === true) && (
+        <ArrowWrapper left={-38}>
+          {props.renderLeftArrow ? (
+            props.renderLeftArrow({
+              Arrow: LeftArrow,
+              getFlickity: flickity,
+              ...props,
+            })
+          ) : (
+            <LeftArrow />
+          )}
+        </ArrowWrapper>
+      )}
+
+      <CarouselContainer height={props.height}>
+        {Flickity && (
+          <Flickity options={settings} ref={flicktyRef}>
+            {props.data.map((slide, index) => {
+              return <Box key={index}>{props.render(slide)}</Box>
```

Comment:
This means that this isn't comparable with SSR and will break the layout. If there's no flickity (no `window`), then `renderSlide` is never called. If that's not called then there will be a big gap when JS is disabled and google indexes content for SEO.